harness/go/history/history.go

```diff
@@ -225,6 +225,8 @@ func scrubRunSpecificScalars(v interface{}) {
 	case *history.ActivityTaskStartedEventAttributes:
 		v.Identity = ""
 		v.RequestId = ""
+		// Disable because cloud may truncate this value
+		v.LastFailure = nil
```

Comment:
Note, this has nothing to do with upgrading the SDK, this is because Cloud changed a config